### PR TITLE
Div 3350 progressbar 2yearsep

### PR DESCRIPTION
**Pull request by borislavpetrovhmcts**
Shows conditional content for AosCompleted state for 2-year separation with  no consent

Fixes # 3350

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested locally, unit test

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: test/unit/steps/petitionerProgressBar.test.js [hunk 1]
```diff
@@ -337,6 +337,39 @@ describe(modulePath, () => {
...
+        state: 'AosCompleted',
+        data: {
+          reasonForDivorce: 'separation-2-years',
+          respAdmitOrConsentToFact: 'No'
+        }
+      }
+    };
+
+    it('renders the correct content', () => {
+      if (!parseBool(config.features.release520)) {
```
Comment (qzhou-hmcts) [inline test/unit/steps/petitionerProgressBar.test.js]: I thought we decided to mock the config for unit test...

File: steps/petition-progress-bar/sections/aosCompleted/PetitionProgressBar.content.aosCompleted.json [hunk 2]
```diff
@@ -5,6 +5,10 @@
     "respDoesNotAdmitAdulteryToMoveForwardYouNeedTo": "To move your divorce forwards you need to either:",
     "respDoesNotAdmitAdulteryChangeReasonForDivorce": "change your reason for divorce",
     "respDoesNotAdmitAdulteryProvideEvidenceOfTheAdultery": "provide evidence of the adultery",
-    "respDoesNotAdmitAdulteryYouCanRead": "You can read your {{ case.divorceWho }}'s response on the next page."
+    "respDoesNotAdmitAdulteryYouCanRead": "You can read your {{ case.divorceWho }}'s response on the next page.",
+    "respDoesNotConsent2YearSepTitle": "Your {{ case.divorceWho }} has not given their consent to a divorce",
+    "respDoesNotConsent2YearSepYouMustHave": "You must have your {{ case.divorceWho }}'s consent to use the '2 years separation with consent' reason for divorce.",
+    "respDoesNotConsent2YearSepYouHaveToPay": "If you want to continue you need to change your reason for divorce. You'll have to pay a fee of £95. Your application will need to be sent to your {{ case.divorceWho }} again.",
```
Comment (gurpiarbassi) [inline steps/petition-progress-bar/sections/aosCompleted/PetitionProgressBar.content.aosCompleted.json]: more of an observation - do we not need to look up the current fee dynamically?

File: test/unit/steps/petitionerProgressBar.test.js (hunk 1 above)
Comment (borislavpetrovhmcts) [reply test/unit/steps/petitionerProgressBar.test.js]: This is to progress the story, Hossain is working on solution for DN, when solution is applied the test will be amended

File: steps/petition-progress-bar/sections/aosCompleted/PetitionProgressBar.content.aosCompleted.json (hunk 2 above)
Comment (borislavpetrovhmcts) [reply steps/petition-progress-bar/sections/aosCompleted/PetitionProgressBar.content.aosCompleted.json]: This was discussed in standup, for this story the fee will be hardcoded, dynamics will be applied later when available in master brunch

**Pull request merged**